src/node_api.cc

```diff
@@ -191,15 +191,15 @@ class Reference : private Finalizer {
     delete reference;
   }
 
-  int Ref() {
+  uint32_t Ref() {
     if (++_refcount == 1) {
       _persistent.ClearWeak();
     }
 
     return _refcount;
   }
 
-  int Unref() {
+  uint32_t Unref() {
     if (--_refcount == 0) {
```

Comment:
Yeah, I wrestled with this myself. Figured the compiler could avoid actually doing two checks of ```_refcount == 0``` if we did check it here, too, but wasn't a big deal either since we only call ```Reference::Unref``` from one place, anyway. I'll just add the extra check.